src/components/add-form/styles.css

```diff
@@ -0,0 +1,24 @@
+.input-item {
+  border-radius: 4px;
+  line-height: 2;
+  margin-bottom: 20px;
+  padding: 0px 10px;
+  width: 100%;
+}
+
+.time-frame-label {
+  margin-bottom: 8px;
+}
+
+.time-frame {
+  display: flex;
+  flex-direction: column;
+  font-size: 15px;
+  margin-bottom: 60px;
+  list-style-type: none;
+  padding: 15px;
+}
```

Comment:
What if you add align-items: flex-start? the items would look like this 
![image](https://user-images.githubusercontent.com/37032132/137219765-f28f5e01-0580-4c2f-b42b-e7e8c9a14a40.png)
 what do you think about this?